### ISSUE TITLE
Accidentally falling halfway can cause the server to crash

### ISSUE THREAD

**Issue by lsz3399747433** [RealAntEngineer/creating_space#2]
When I was playing multiplayer games Falling in moon_orbit and earth_orbit  will cause the server to crash

**Comment by lsz3399747433** [comment]
I really like this module, and I feel that the future is promising, but I have PlotSquared on my server, and I hope it can be used in that world and not limited to the main world

**Comment by RealAntEngineer** [comment]
Can you give the list of mod installed and the log/ debuglog/ crash report ?

**Comment by lsz3399747433** [comment]
It's working now, but can I change it to be usable in another world?

**Comment by RealAntEngineer** [comment]
Can you discribe a little more what you whant to do ?

**Comment by lsz3399747433** [comment]
Hello, for example, in multiplayer games, I have many worlds. I don't want to use it only in the initial 'world' but there's also another issue when I use this mod, which is logged. I'm not sure if it's a problem with this mod.


mod list:createhaven, creatingspace, salwayseat, tonsofenchants, playeranimator, tanukidecor, jei, obscure_api, immersive_aircraft, sophisticatedcore, create_jetpack, cookingforblockheads, citadel, alexsmobs, artifacts, sereneseasons, mixinextras, bookshelf, create_confectionery, sophisticatedbackpacks, create_dragon_lib, createdieselgenerators, farsight_view, balm, terralith, cloth_config, forge, mohist, ctov, dungeons_arise, minecraft, cofh_core, thermal, thermal_integration, farmersdelight, casualness_delight, terrablender, bettercombat, thermal_innovation, avaritia, biomesoplenty, jade, ae2, creativecore, nethersdelight, astikorcarts, kotlinforforge, curios, mekanism, mekanismgenerators, cataclysm, touhou_little_maid, iceberg, flywheel, create, create_central_kitchen, create_mechanical_extruder, create_sa, searchables, storagedrawers, immersive_paintings, thermal_expansion, thermal_foundation, ftbultimine, thermal_locomotion, constructionwand, butchersdelight, seasonhud, architectury, ftblibrary, doapi, terraform, candlelight, bakery, vinery, ftbteams, ftbchunks, yuushya, lootr, chimes, cupboard, flightlib, jadeaddons, packetfixer, cosmeticarmorreworked, expandability, chunksending, thermal_dynamics, createaddition, drinkbeer, cristellib

Here is the log of my crash. Thank you.

[latest.log](https://github.com/RealAntEngineer/creating_space/files/14142819/latest.log)

**Comment by RealAntEngineer** [comment]
There is an issue with distant server right now, it's solved in the dev version so in 1.7.1 will be solved. I think that was the cause of the crash when falling. otherwise I don't see anything related to my mod in the log

**Comment by RealAntEngineer** [comment]
also can you give me the crash report ?

**Comment by RealAntEngineer** [comment]
Can you try with the new 1.7.0 version ? the bug should be solved.

**Comment by lsz3399747433** [comment]
Thank you so much for your help; I don't have any further issues at the moment. However, I do have a suggestion. I'm using this mod on a Minecraft server, and because I've set up multiple worlds, I hope its functionality isn't limited to just taking off in the 'world' world (which is the main world). I'd like to use it in other worlds as well. For instance, in a world named 'survival', just to give an example. I hope there's a feature that allows for such customization.

**Comment by lsz3399747433** [comment]
![image](https://github.com/RealAntEngineer/creating_space/assets/154571421/69d2970d-2dc6-40fc-bda2-2de530bb1680)
"Hello, when I used version 1.7.0b, I noticed particle effects but there was no takeoff, and then the following appeared in the console:
[18:48:26 INFO]: finishing setting up parameters
[18:48:26 WARN]: Received physics frame update for ship with ShipId: 76, but a ship with this ShipId does not exist!
[18:48:26 WARN]: Received physics frame update for ship with ShipId: 76, but a ship with this ShipId does not exist!
[18:48:26 WARN]: Received physics frame update for ship with ShipId: 76, but a ship with this ShipId does not exist!
[18:48:26 WARN]: Received physics frame update for ship with ShipId: 76, but a ship with this ShipId does not exist!
[18:48:41 INFO]: finishing setting up parameters
[18:48:41 WARN]: Received physics frame update for ship with ShipId: 77, but a ship with this ShipId does not exist!
